Utils/Extensions/NSURLExtensions.swift

```diff
@@ -223,6 +223,10 @@ extension NSURL {
 
         return false
     }
+
+    public var isLocal: Bool {
+        return host == "localhost" || host == "127.0.0.1"
```

Comment:
Also needs `::1` for IPv6 localhost. And do we know for sure that `http://LOCALHOST/` gets turned into a lowercase `localhost` ?